src/touchstone/databases/prometheus.py

```diff
@@ -0,0 +1,43 @@
+import dateparser
+from prometheus_api_client import PrometheusConnect
+
+
+class Prometheus:
+    def __init__(self, query_list, start_time_list, end_time_list,
+                 url="http://127.0.0.1:9090", headers=None, disable_ssl=False):
+        self.url = url
+        self.headers = headers
+        self.disable_ssl = disable_ssl
+        self.query_list = query_list
+        self.start_time_list = start_time_list
+        self.end_time_list = end_time_list
+        self.pc = PrometheusConnect(url=url, headers=headers, disable_ssl=disable_ssl)
+
+    def compare_data(self):
+        output = []
+        len_times = 1 if self.start_time_list is None else len(self.start_time_list)
+        for query in self.query_list:
+            for i in range(len_times):
+                aggregates = {'url': self.url, 'query': query}
+                settings = {"DATE_ORDER": "YMD"}
+                if self.start_time_list is not None:
+                    aggregates['start_time'] = self.start_time_list[i]
+                    start_time = dateparser.parse(str(self.start_time_list[i]),
+                                                  settings=settings)
+                else:
+                    start_time = None
+                if self.end_time_list is not None:
+                    aggregates['end_time'] = self.end_time_list[i]
+                    end_time = dateparser.parse(str(self.end_time_list[i]), settings=settings)
```

Comment:
Why do you use dateparser?, you can get a similar result with timedate. i.e `datetime.datetime.fromtimestamp(1588795118)`, which is in the python  standard library.